docs/content/Getting-Started-Docker.md

```diff
@@ -29,26 +26,48 @@ services:
       # 3000 is a port for Playground web server
       # it is available only in dev mode
       - 3000:3000
-    env_file: .env
     volumes:
       - ./schema:/cube/conf/schema
 ```
 
 ## 2. Configure Cube.js
 
+<!-- prettier-ignore-start -->
+[[info |]]
+| This step assumes you can connect to a database instance. If you're unable
```

Comment:
How about rephrasing this to something like the following? 

This step assumes that you have the necessary credentials to access a database instance. If you are unable to connect to a database instance, please use a Docker image to run a local database in your development environment."

Reply:
Reads much better, thanks @rpaik 👍